batchflow/pipeline.py

```diff
@@ -1239,13 +1240,13 @@ def rebatch(self, batch_size, fn=None, components=None, batch_class=None):
         return new_p._add_action(REBATCH_ID, _args=dict(batch_size=batch_size, pipeline=self, fn=fn,
                                                         components=components, batch_class=batch_class))
 
-    def _put_batches_into_queue(self, gen_batch, bar, bar_desc):
+    def _put_batches_into_queue(self, gen_batch, bar):
         while not self._stop_flag:
             self._prefetch_count.put(1, block=True)
             try:
                 batch = next(gen_batch)
                 if bar:
```

Comment:
Let's always create notifier and always call `notifier.update()`... sometimes it will do nothing